lms/models/lti_user.py

```diff
@@ -28,6 +28,12 @@ class LTIUser:  # pylint: disable=too-many-instance-attributes
     application_instance_id: int
     """ID of the application instance this user belongs to"""
 
+    context_id: str
```

Comment:
The course and assignment ids (lms ids?) are needed in lots of API queries.

We often send them in json as part of the data provided in JS config to the frontend and then read them back. 

I reckon LTIUser is a good place for them because:

- They make sense here, LTIUser is built from a LTI launch and there's no launch without a context. Deep linking launches don't have an assignment linked to them so that's optional.

- LTIUser is used in all be/fe communication. No need to add it to js config or make changes to frontend code.

- LTIUser is signed. LTIUser is used for authentication and it's signed so it can't be tampered with. I reckon a motivated user could change values in other API calls and we'd accepted them without question.


That being said I want to make clear that:

- LTIUser should not become a bag for random values. These make sense and are very central to LTI launches. A LTI launch provides info about who you are and in which context. LTIUser now reflects that.

- We should provided a better mechanism to share info between fe/be. At least we should sign all parameters indepently of changes to LTIUser.